core/modules/menu/js/menu.admin.js

```diff
@@ -19,6 +19,60 @@ Backdrop.behaviors.menuAdminFieldsetSummaries = {
   }
 };
 
+Backdrop.behaviors.menu_edit_item_parents = {
```

Comment:
Objects in JavaScript we tend to use snake case:

```suggestion
Backdrop.behaviors.menuEditItemParents = {
```

Reply:
Fixed